app/models/notifications/NotificationDetails.scala

```diff
@@ -33,8 +35,8 @@ case class NotificationDetails(contactType: Option[ContactType],
 
   def subject(templateVersion: String) =
     templateVersion match {
-      case "v1m0" | "v2m0" | "v3m0" | "v4m0" => s"notifications.subject.$cType"
-      case "v5m0" => s"notifications.subject.v5.$cType"
+      case "v5m0" if(isContactTypev5(cType))=> s"notifications.subject.v5.$cType"
```

Comment:
Instead of adding new message for all contact TYpe, I have added a logic that new heading applicable to only those contact types whose templates have changed otherwise use the old heading from message file.